src/main.js

```diff
@@ -7,20 +7,23 @@ import { subscribe } from '@nextcloud/event-bus'
 import { loadState } from '@nextcloud/initial-state'
 
 /**
- * - Expose OCA.TPAssistant.openTextProcessingModal to let apps use the assistant
+ * - Expose OCA.Assistant.openAssistantForm to let apps use the assistant
  * - Add a header right menu entry
  * - Listen to notification event
  */
 function init() {
-	if (!OCA.TPAssistant) {
+	if (!OCA.Assistant) {
 		/**
 		 * @namespace
 		 */
-		OCA.TPAssistant = {
+		OCA.Assistant = {
```

Comment:
Are there docs anywhere that need to be changed as well?

Reply:
Working on it :grin: